docs/TFM/projectTool/Walkthrough.md

```diff
@@ -170,6 +170,22 @@ make sure you have created the files:
 - alive_rv32i_elf.txt  
 - alive_inst_mem_rv32i.sv  
 
+### Running RISCV toolchain using GUI
```

Comment:
Again - seems like we are mixing here 2 things..
1) the tools we use (which is not dependent on the mafia)
2) the mafia flow (which is using the existing tools.. but has its own scripts and TFM)